.github/workflows/nodejs.yaml

```diff
@@ -26,10 +29,19 @@ jobs:
         with:
           envkey_FAUNA_DB_DEV: ${{ secrets.FAUNA_DB_DEV }}
       - run: npm ci
+      # todo: if FAUNA_DB_DEV isn't set, don't run the fauna tests
```

Comment:
I'll add this in a follow-up PR! Just doing one thing at a time. We need to make sure this works when we merge to master, first.